Microsoft.Toolkit.Graph.Controls/Controls/LoginButton/LoginButton.cs

```diff
@@ -181,6 +180,12 @@ public async Task LoginAsync()
         /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
         public async Task LogoutAsync()
         {
+            // Close Menu
```

Comment:
Actually wondering if we should just put this in the [sign-out button event](https://github.com/windows-toolkit/Graph-Controls/pull/45/files#diff-e556fd96bc615fd2f6458f2a2dee2feaR101)?

Though, I'm trying to think of if we care about keeping the flyout open if the developer decides to cancel the logout for some reason from the `LogoutInitiated` event. @nmetulev / @beth-panx / @shweaver-MSFT is this something you've thought of/encountered yet in the MGT?

Reply:
I had the same question and I ended putting it here to be able to hide the flyout if the `LoqoutAsync()` method is called from the code behind while the flyout is opened.